data/styles/vehicle/include/priorities_4_overlays.prio.txt

```diff
@@ -229,20 +229,19 @@ shop-caravan                                        # icon z17-
 
 amenity-parking                                     # icon z15- (also has caption(optional) z16-, area z15-)
 amenity-parking-fee                                 # icon z15- (also has caption(optional) z16-, area z15-)
+amenity-parking-fee-underground                     # icon z16- (also has caption(optional) z16-)
 amenity-parking-multi-storey                        # icon z15- (also has caption(optional) z16-, area z15-)
 amenity-parking-park_and_ride                       # icon z15- (also has caption(optional) z16-, area z15-)
 amenity-parking-permissive                          # icon z15- (also has caption(optional) z16-, area z15-)
-amenity-parking-underground                         # icon z15- (also has caption(optional) z16-)
+amenity-parking-underground                         # icon z16- (also has caption(optional) z16-)
```

Comment:
why make it less visible than regular parkings?

Reply:
Fixed. Before it was showing as regular parking at z15 and as underground parking from z16 onwards. As fixed it will show as underground parking from z15-